api.go

```diff
@@ -16,43 +16,112 @@
 package brokerapi
 
 import (
+	"github.com/pivotal-cf/brokerapi/v11/auth"
+	"github.com/pivotal-cf/brokerapi/v11/domain"
+	"github.com/pivotal-cf/brokerapi/v11/handlers"
```

Comment:
thoughts: standard packages before that internal packages

Reply:
I've updated this. We could run `goimports` on the whole codebase, but I'd rather do that in a different PR.